truthful.py

```
@@ -0,0 +1,19 @@
+import os
+from ast import literal_eval
+
+if __name__ == "__main__":
+    for val in sys.argv[1:]:
+        try:
+            val = literal_eval(val)
+        except (ValueError, SyntaxError):
+            try:
+                val = json.loads(val)
+            except ValueError:
+                pass  # keep it a string
```

Ruff (0.12.2)

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

(SIM105)